src/gvpn_controller.py

```diff
@@ -24,17 +25,18 @@ def __init__(self, user, password, host, ip4):
                 self.uid_ip_table[uid] = ip
 
         if CONFIG["icc"]:
-            self.inter_controller_conn()
+            self.inter_controller_conn() # UDP Server for Inter Controller Connection.
         
         if CONFIG["switchmode"]:
             self.arp_table = {}
 
+        # Ignore the network interfaces in the list.
         if "network_ignore_list" in CONFIG:
             logging.debug("network ignore list")
             make_call(self.sock, m="set_network_ignore_list",\
                              network_ignore_list=CONFIG["network_ignore_list"])
 
-
+    #Initialise the Controller
```

Comment:
Put a space after the `#`, and s/Initialise/Initialize/

Also (and this applies in multiple places), if you find yourself putting comments above a function or class, instead consider putting them as [docstrings](https://www.python.org/dev/peps/pep-0257/#specification). Docstrings are a special language feature that allows other forms of documentation to be generated from the source code later.